src/components/Clips/ClipsPortal.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+
+import {
+    Typography,
+    makeStyles,
+    Divider,
+    List,
+    ListItem,
+    Chip,
+} from '@material-ui/core';
+import Thumbnail from '../../assets/default-thumbnail.jpg';
+import { ClipData } from '.';
+
+const useStyles = makeStyles(() => ({
+    root: {},
+    thumbnail: {
+        maxWidth: '100%',
+        height: 'auto',
+    },
+    links: {
+        display: 'flex',
+        flexDirection: 'column',
+        rowGap: '10px',
+    },
+    chips: {
+        display: 'flex',
+        columnGap: '2px'
+    }
+}));
+
+interface Props {
+    clip: ClipData;
+}
+
+export default function ClipsPortal({ clip }: Props) {
+    const classes = useStyles();
+
+    return (
+        <section className={classes.root}>
+            <List className={classes.links}>
+                <img
+                    alt='Clip Thumbnail'
+                    className={classes.thumbnail}
+                    src={Thumbnail}
+                />
+                <div className={classes.chips}>
+                    {clip.tags.map((tag, index) => {
+                        return <Chip color='primary' clickable size='small' label={tag} />;
+                    })}
+                </div>
+                <Typography variant='h4' gutterBottom>
+                    Mark Takano
```

Comment:
We'll need to add more props to this so that way this can be data fetched from the server.